<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSGoodsBigCategoryWidget</class>
 <widget class="QWidget" name="BCSGoodsBigCategoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="OrderCategoryBigListView" name="listView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView{
	border:none;
	background-color: rgb(81, 80, 80);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>77</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QPushButton" name="lastPageButton">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>16</y>
        <width>15</width>
        <height>18</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>18</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>18</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-image: url(:/setImage/left.png);
}

QPushButton:pressed{
	border-image: url(:/setImage/left_press.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>38</x>
        <y>16</y>
        <width>1</width>
        <height>18</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>18</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>18</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #ededed;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPageButton">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>16</y>
        <width>15</width>
        <height>18</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>18</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>18</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image: url(:/setImage/right.png);
}

QPushButton:pressed{
border-image: url(:/setImage/right_press.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrderCategoryBigListView</class>
   <extends>QListView</extends>
   <header>OrderCategoryBigListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
